package.json

```diff
@@ -80,7 +80,7 @@
     "@equinor/fusion-framework-react-app": "^2.0.30",
     "@equinor/fusion-framework-react-module-context": "^1.0.17",
     "@equinor/workspace-ag-grid": "^0.3.5",
-    "@equinor/workspace-fusion": "^0.15.4",
+    "@equinor/workspace-fusion": "^0.16.1",
```

Comment:
Any breaking changes not related to PBI in this new version?

Reply:
Nope